rrmngmnt/host.py

```diff
@@ -336,3 +337,41 @@ def is_connective(self, tcp_timeout=20.0):
             ret = False
 
         return ret
+
+    def download_file_from_url(self, url, f_dir):
```

Comment:
maybe use  https://pypi.python.org/pypi/wget 
and name can be only download_file()